### PR TITLE
MC-12631: Add delete cluster API documentation

### PR DESCRIPTION
**Pull request by HassanHaidar**
#### Changes made

- Update documentation with Delete Cluster operation

#### Related PRs
- [#357](https://github.com/cloudops/cloudmc-gcp-plugin/pull/357)

### CODE REVIEW

**Review by patrickspensieri — COMMENTED** [review]
Looks good! Usually we add the expected response, in this case guessing it looks like this

```
> The above command returns a JSON structured like this:

```json
{
  "taskId": "3e4d4466-ce4b-404b-ada5-ee5a3fb76f4e",
  "taskStatus": "PENDING"
}
```

**Comment by HassanHaidar** [comment]
> Looks good! Usually we add the expected response, in this case guessing it looks like this
> 
> ```
> > The above command returns a JSON structured like this:
> 
> ```json
> {
>   "taskId": "3e4d4466-ce4b-404b-ada5-ee5a3fb76f4e",
>   "taskStatus": "PENDING"
> }
> ```

5439db98fc9d271320fbee3f325ce4c770e9dfe0

File: source/includes/gcp/_clusters.md [hunk 1]
```diff
@@ -196,3 +196,25 @@ Required | &nbsp;
...
+
+```shell
+curl -X DELETE \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/gcp/test-area/clusters/projects/cmc-test-area-ugx/locations/us-central1-c/clusters/cluster-1"
+```
+
+<code>DELETE /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/clusters/:id</code>
+
+Delete a cluster. A cluster can only be deleted if it is in RUNNING or ERROR states.
```
Comment (GuillaumeVialle) [inline source/includes/gcp/_clusters.md]: state

Comment (HassanHaidar) [reply source/includes/gcp/_clusters.md]: 3007be739915ff02a28e2b0e45beca95a6db0a96

**Review by GuillaumeVialle — APPROVED** [review]
LGTM

**Review by patrickspensieri — APPROVED** [review]

**Pull request merged**